webapp/WEB-INF/templates/admin/dashboard/admin/search_admindashboard.html

```diff
@@ -46,7 +46,7 @@
         </@row>
         <!-- HELP MESSAGE -->
         <@formGroup labelFor='help_message' labelKey='${i18n("portal.search.advanced_parameters.helpMessage.label")}'>
-            <@input type='textarea' class='richtext' name='help_message' id='help_message' value=help_message!'' />
+            <@input type='textarea' name='help_message' id='help_message' value=help_message!'' />

```

Comment:
please remove also the tags  <#include "/admin/util/editor/editor.html" /> and <@initEditor />